feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,382 @@
+
+  # State Paths:
+  # Folliwing for normal health check pre and post each test
+  /system/processes/process/state/cpu-utilization
+  /system/processes/process/state/memory-utilization
+  /qos/interfaces/interface/input/queues/queue/state/dropped-pkts
+  /qos/interfaces/interface/output/queues/queue/state/dropped-pkts
+  /qos/interfaces/interface/input/virtual-output-queues/voq-interface/queues/queue/state/dropped-pkts
+  /interfaces/interface/state/counters/in-discards
+  /interfaces/interface/state/counters/in-errors
+  /interfaces/interface/state/counters/in-multicast-pkts
+  /interfaces/interface/state/counters/in-unknown-protos
+  /interfaces/interface/state/counters/out-discards
+  /interfaces/interface/state/counters/out-errors
+  /interfaces/interface/state/oper-status
+  /interfaces/interface/state/admin-status
+  /interfaces/interface/state/counters/out-octets
+  /interfaces/interface/state/description
+  /interfaces/interface/state/type
+  /interfaces/interface/subinterfaces/subinterface/state/counters/in-discards
+  /interfaces/interface/subinterfaces/subinterface/state/counters/in-errors
+  /interfaces/interface/subinterfaces/subinterface/state/counters/in-unknown-protos
+  /interfaces/interface/subinterfaces/subinterface/state/counters/out-discards
+  /interfaces/interface/subinterfaces/subinterface/state/counters/out-errors
+  /interfaces/interface/ethernet/state/counters/in-mac-pause-frames
+  /interfaces/interface/ethernet/state/counters/out-mac-pause-frames
+  /interfaces/interface/ethernet/state/counters/in-crc-errors
+  /interfaces/interface/ethernet/state/counters/in-block-errors
+
+  # IPv4oUDP Decap state
+  /network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/state/decapsulate-gue
+
+  # BGP state paths
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/session-state
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/received-prefixes
+  /network-instances/network-instance/protocols/protocol[identifier='BGP']/bgp/neighbors/neighbor/state/advertised-prefixes
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/peer-as
+
+  # Packet counters matching forwarding rules for IPv4oUDP Decap
+  /network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/state/counters/forwarded-packets
+  /network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/state/counters/dropped-packets
+  /network-instances/network-instance[name='<instance-name>']/policy-forwarding/state/active-policy
+  /network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/action/state/next-hop
+
+
+
+rpcs:
+  gnmi:
+    gNMI.Set:
+      union_replace: true
+    gNMI.Subscribe:
+      on_change: true
+```
+
+## Required DUT platform
+
+* Specify the minimum DUT-type:
```

Comment:
Again, I think this is not the minimum -- must be MFF since MFF > FFF if you need MFF, but I'm not sure why this can't be executed against FFF.